### ISSUE TITLE
Dedicated keyboard landing pages need to be updated for mobile and other platforms

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman.com#174]
Right now, most of the dedicated landing pages still focus on Windows, although the keyboards contained mostly work on all platforms. Furthermore, we have not really tested them for responsive layout.

This should be completed prior to 14.0 release, because the landing pages will appear in the keyboard search results in embedded searches.

**Comment by darcywong00** [comment]
template.css might be making the keyboard landing page wonky
```css
.download-cta-big-p, .download-cta-browser p{
    float: left;
    width: 40% <----
```

**Comment by mcdurdin** [comment]
I suspect there will be a number of changes required to the CSS to make them work... Is this something you are comfortable tackling @darcywong00?

**Comment by darcywong00** [comment]
Fixed by #248
